cognite/client/data_classes/simulators/runs.py

```diff
@@ -424,10 +440,56 @@ def dump(self, camel_case: bool = True) -> dict[str, Any]:
         output["outputs"] = [output_.dump(camel_case=camel_case) for output_ in self.outputs]
         return output
 
+    def to_pandas(  # type: ignore [override]
+        self,
+    ) -> pandas.DataFrame:
+        """Convert the simulation run data to a pandas DataFrame.
+
+        Returns:
+            pandas.DataFrame: The dataframe.
+        """
+        pd = local_import("pandas")
+
+        def _create_row(item: SimulationInput | SimulationOutput) -> dict:
+            """Create a row dictionary for an input or output item."""
+            item_type = (isinstance(item, SimulationInput) and "Input") or "Output"
+            row = {
+                "run_id": self.run_id,
+                "type": item_type,
+                "reference_id": item.reference_id,
+                "value": item.value,
+                "unit_name": item.unit.name if item.unit else None,
+                "value_type": item.value_type,
+                "overridden": getattr(item, "overridden", None),
+                "timeseries_external_id": item.timeseries_external_id,
+            }
+
+            if item.simulator_object_reference:
+                for reference_key, reference_value in item.simulator_object_reference.items():
+                    snake_key = to_snake_case(reference_key)
+                    row[snake_key] = reference_value
+
+            return row
+
+        rows = [_create_row(item) for item in self.inputs + self.outputs]
+
+        return pd.DataFrame(rows)
+
 
 class SimulatorRunDataList(CogniteResourceList[SimulationRunDataItem], IdTransformerMixin):
     _RESOURCE = SimulationRunDataItem
 
+    def to_pandas(  # type: ignore [override]
+        self,
+    ) -> pandas.DataFrame:
```

Comment:
The goal of overwriting here is to simplify the signature?

Reply:
I just followed what others done, but yeah, the base implementation has all these props which we don't need
```
    def to_pandas(
        self,
        camel_case: bool = False,
        expand_metadata: bool = False,
        metadata_prefix: str = "metadata.",
        convert_timestamps: bool = True,
    ) -> pandas.DataFrame:
```